src/emulate.c

```diff
@@ -604,16 +619,18 @@ static void block_translate(riscv_t *rv, block_t *block)
     block->pc_start = block->pc_end = rv->PC;
 
     rv_insn_t *prev_ir = NULL;
-    rv_insn_t *ir = mpool_calloc(rv->block_ir_mp);
+    rv_insn_t *ir = mpool_alloc(rv->block_ir_mp);
```

Comment:
Why should mpool_calloc() be replaced with mpool_alloc() and then execute memset()?

Reply:
It seems like a rebase problem for #437.